hal/lib/shared/HAL.cpp

```diff
@@ -18,17 +18,22 @@ struct HAL_JoystickAxesInt {
 
 extern "C" {
 
-int HAL_GetControlWord(HAL_ControlWord* data) {
-  return FRC_NetworkCommunication_getControlWord((ControlWord_t*)data);
+HAL_ControlWord HAL_GetControlWord() {
```

Comment:
I think I prefer the pointer taking versions as well. Easier to hold caches of data when necessary.